data/json/recipes/recipe_deconstruction.json

```diff
@@ -5810,5 +5810,13 @@
     "difficulty": 3,
     "qualities": [ { "id": "FABRIC_CUT", "level": 2 } ],
     "components": [ [ [ "nylon", 6 ] ] ]
+  },
+  {
+    "result": "towel",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "30 m",
```

Comment:
30 minutes is a *long* time to cut a towel into rags. Perhaps something shorter?

Reply:
I have shortened it to five minutes.